src/services/identity/postgres.ts

```diff
@@ -301,14 +328,27 @@ export class PostgresIdentityService extends DefaultIdentityService {
 		return identifier;
 	}
 
-	async createResource(network: string, payload: ResourcePayload, customer: CustomerEntity) {
+	async createResource(
```

Comment:
Shouldn't the local strategy be updated accordingly as well?

That's relevant for at least DLR creation, DID deactivation, ensuring consistent behavior on the intersection of inherently implemented or not methods.